src/software/ai/hl/stp/tactic/move/simulated_move_tactic_test.cpp

```diff
@@ -42,6 +44,13 @@ TEST_F(SimulatedMoveTacticTest, test_move_across_field)
                 yield("Tactic not done");
             }
             robotAtPosition(1, world_ptr, destination, 0.05, yield);
+            auto stopped_time = world_ptr->getMostRecentTimestamp();
+            while (world_ptr->getMostRecentTimestamp() <
+                   stopped_time + Duration::fromSeconds(3))
+            {
+                yield("Waiting 3 seconds before re-checking conditions");
```

Comment:
Why not just check on each tick? Whatever the reason, we should add a comment explaining why we do this. Same goes for similar delays in other tests.

Reply:
checking every tick now